src/test/java/org/prebid/server/bidder/smaato/SmaatoBidderTest.java

```diff
@@ -184,6 +185,33 @@ public void makeHttpRequestsShouldModifyRequestSite() {
                 .containsOnly("keywords");
     }
 
+    @Test
+    public void makeHttpRequestsShouldModifyRequestApp() {
+        // given
+        final BidRequest bidRequest = BidRequest.builder()
+                .imp(singletonList(Imp.builder()
+                        .id("123")
+                        .banner(Banner.builder()
+                                .id("banner_id").format(asList(Format.builder().w(300).h(500).build(),
+                                        Format.builder().w(450).h(150).build())).build())
+                        .ext(mapper.valueToTree(ExtPrebid.of(null,
+                                ExtImpSmaato.of("publisherId", "adspaceId")))).build()))
+                .app(App.builder().build())
+                .build();
```

Comment:
Can be simplified, if you add `.w(300).h(500)` to givenImp() banner object,
than u can use 
```java
final BidRequest bidRequest = givenBidRequest(bidRequestBuilder ->
                bidRequestBuilder.app(App.builder().build()), identity());
```

Reply:
Done